### ISSUE TITLE
Bubble layout does not enlarge emojis

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#20660]
### Steps to reproduce

1. Enable bubble layout
2. Look at any message containing emojis

### Outcome

![Screenshot 2022-01-20 at 23-34-22 Element Test room](https://user-images.githubusercontent.com/48614497/150466433-65544a19-79c6-4edb-9036-052332469d8b.png)

#### What did you expect?

The emoji should be enlarged, as in the modern layout.

#### What happened instead?

The emoji is the same size as the rest of the text.

### Operating system

NixOS unstable

### Browser information

Firefox 96.0

### URL for webapp

develop.element.io

### Application version

Element version: c40dbf158019-react-2e6f616e91f6-js-230e3b4acee3 Olm version: 3.2.8

### Homeserver

townsendandsmith.ml

### Will you send logs?

No

**Comment by t3chguy** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/7602
